stable/rabbitmq-ha/values.yaml

```diff
@@ -253,6 +253,15 @@ rabbitmqWebSTOMPPlugin:
     # web_stomp.ws_frame = binary
     # web_stomp.cowboy_opts.max_keepalive = 10
 
+## Prometheus Plugin
+## Ref: https://www.rabbitmq.com/prometheus.html
+rabbitmqPrometheusPlugin:
+  enabled: false
+  ## Prometheus configuration:
+  config: |
```

Comment:
I don't see how this config is used anywhere.

Reply:
sorry you are right, fixed here https://github.com/helm/charts/pull/17925/commits/4ff4bf79367310d2f7e43ae8a997ee57fda7064b#diff-28e0ed5d78530ec2e1cf87dab1eb62a5R115-R119